webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/AccessTokenService.scala

```diff
@@ -20,7 +20,7 @@ object AccessMode extends ExtendedEnumeration {
 
 object AccessResourceType extends ExtendedEnumeration {
   type AccessResourceType = Value
-  val datasource, tracing, annotation, webknossos, jobExport = Value
+  val datasource, dataset, tracing, annotation, webknossos, jobExport = Value
```

Comment:
hmm again naming things is hard :grimacing: I don’t really have a clean solution either. I hope we can just get rid of the traditional/datasource codepaths soon.